package.json

```diff
@@ -32,9 +32,8 @@
     "tailwindcss": "^3.2.3"
   },
   "scripts": {
-    "build": "node esbuild.config.js",
-    "build:css": "bin/link; yarn light:build; yarn light:build:css; yarn light:build:mailer:css",
-    "light:build": "esbuild `bundle exec bin/theme javascript light` --bundle --sourcemap --outdir=app/assets/builds --loader:.png=file --loader:.jpg=file",
+    "build": "THEME=\"light\" node esbuild.config.js",
+    "build:css": "bin/link; THEME=\"light\" yarn build; yarn light:build:css; yarn light:build:mailer:css",
```

Comment:
@KonnorRogers, I'm sorry to comment on such an old PR, but I'm wondering if you might happen to remember why you added `yarn build` as a part of `yarn build:css` here. It results in `yarn build` being run twice when we call `rails assets:precompile`. It gets called once directly by the `precompile` task, and then gets called a second time as part of `build:css`.

Reply:
No worries. Umm...oversight maybe? I can't remember if there was some issue with needing to build in the CSS because of race conditions in the Procfile...it's been a while 😅

Reply:
Either that, or I just blindly followed what we there before and didn't realize it...really not sure tbh...